pkg/common/telemetry/inmem.go

```diff
@@ -51,16 +64,45 @@ func (i *inmemRunner) run(ctx context.Context) error {
 	}
 
 	var wg sync.WaitGroup
+
+	i.startInMemMetrics(ctx, &wg)
+
+	if !i.inMemBlockSet {
+		i.startConfigWarning(ctx, &wg)
+	}
+
+	wg.Wait()
+	return nil
+}
+
+func (i *inmemRunner) startConfigWarning(ctx context.Context, wg *sync.WaitGroup) {
+	wg.Add(1)
+	sigChannel := make(chan os.Signal, 1)
+	signal.Notify(sigChannel, metrics.DefaultSignal)
+	fmt.Println("setting up")
```

Comment:
debugging cruft?

Reply:
I suppose I should actually `save` in my editor before committing :) oops